packages/frontend-web/src/app/components/OverflowContainer/index.ts

```diff
@@ -14,4 +14,4 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-export { OverflowContainer } from './OverflowContainer';
+export * from './OverflowContainer';
```

Comment:
Suggest that we use named exports instead of *?

Reply:
Hi Jeffrey,  thanks for the review.  Want me to include you as a reviewer on all subsequent reviews?  

WRT this item, I don't use default exports, if I can help it.  In particular, all the exports from OverflowContainer are named.  

This line is just reexporting everything from OverflowContainer.  We use this idiom quite a lot.  But if you know of a better way of doing it, let me know.

Speak soon,

Reply:
Yes you can send reviews to me, while I lack Lucas' depth of knowledge about typescript, I usually respond fast.